frontend/src/services/requestService.js

```diff
@@ -9,18 +10,14 @@ axiosInstance.interceptors.request.use((config) => {
     config.headers['Content-type'] = 'application/json';
     config.headers['Accept'] = 'application/json';
 
-    const token = getToken();
-
-    if (token) {
-        config.headers.Authorization = token;
+    if (storage.hasToken()) {
+        config.headers.Authorization = 'Bearer ' + storage.getToken();
     }
 
     return config;
 });
 
-const getToken = () => {
-    return 'Bearer ' + 'tokenFromSessionStore';
-};
+axiosInstance.interceptors.response.use(response => response.data.data);
```

Comment:
Better remove this, or change just to `response.data` because otherwise no one will be able to get meta info from response